marble/marble.go

```diff
@@ -66,8 +66,8 @@ func generateFromEnv() (tls.Certificate, *x509.CertPool, error) {
 	}
 
 	roots := x509.NewCertPool()
-	if !roots.AppendCertsFromPEM(intermediateCA) {
-		return tls.Certificate{}, nil, fmt.Errorf("cannot append intermediateCA as rootCA to CertPool")
+	if !roots.AppendCertsFromPEM(rootCA) {
```

Comment:
Rename this to marbleRootCA, maybe?

Otherwise might be confusing when working between Coordinator <-> Marble code?